jekyll/_cci2/github-integration.adoc

```diff
@@ -60,6 +60,7 @@ CircleCI requests the following permissions from GitHub, defined in the link:htt
 
 - Get a list of a user's repos
 - Add an SSH key to a user's account
+- Add Environment Variables to Project Settings
```

Comment:
So my understanding is this is a list of thing CircleCI can do on a user's behalf once they have these permissions. Rather than listing things the user is able to do. Maybe there's a better way to surface this 🤔 Does that make sense?

Reply:
@rosieyohannan - apologies for delay on this!

Totally makes sense and I think you're correct that this isn't the best way to surface this as this is more of what _CircleCI can do on behalf of user_

We have an equivalent for Apps/GL but not exactly for OAuth integrations and I'm thinking an equivalent would be the best approach for this:

> https://circleci.com/docs/roles-and-permissions-overview/

Reply:
Oh interesting! Thanks, I'll turn this into a Jira and then close the PR